app/enrichments/remove_placeholder.rb

```diff
@@ -0,0 +1,53 @@
+##
+# Removes placeholder values from a field. Applies to both straight string 
+# values and Resources with a matching `providedLabel`. Matches are case
+# insensitive.
+#
+# @example 
+#   rp = RemovePlaceholder.new('moomin')
+#   rp.enrich_value('moomin') # => nil
+#   rp.enrich_value('MooMin') # => nil
+#   rp.enrich_value('Little My') # => 'Little My'
+#
+# @example with Resource value
+#   rp = RemovePlaceholder.new('moomin')
+#   concept = DPLA::MAP::Concept.new
+#   concept.providedLabel = 'moomin'
+#   rp.enrich_value(concept) # => nil
+#
+# @see Audumbla::FieldEnrichment
+class RemovePlaceholder
+  include Audumbla::FieldEnrichment
+
+  ## 
+  # @param [String] placeholder  a string value to regard as a placeholder
+  def initialize(placeholder = 'xyz')
```

Comment:
See comment on [line 49](https://github.com/dpla/heidrun/pull/34/files#r40973002). We'd probably want a `downcase = true` or something like that if we want to add the flexibility